src/components/index.js

```diff
@@ -0,0 +1,4 @@
+import { Navbar } from "./navbar/Navbar";
+import { Carousel } from "./carousel/Carousel";
```

Comment:
You can just export and use it.
export { Navbar } from "./navbar/Navbar";

Reply:
done